packages/visual-editor/src/types/constants.ts

```diff
@@ -12,7 +12,7 @@ export const CTFL_DRAGGABLE_ID = 'data-ctfl-draggable-id';
 export const CTFL_DRAGGING_ELEMENT = 'data-ctfl-dragging-element';
 
 export const HITBOX = {
-  WIDTH: 80,
+  WIDTH: 70,
```

Comment:
Is this a change we should discuss with the team about async in slack? Or maybe at least an FYI thing - two main modes of consideration I have here

1. This is a fundamental change to every component hence why it should be at least an FYI
2. While it's an impactful change, it's a minor difference numerically and is an easy two way street. So I think anything longer than a quick discussion would result in negative return on value relative to time spent.

Reply:
Yeah good point, I can make a quick note in Slack. It's also just for the invisible hitboxes so no change in terms of the visible UI but does lead to a change in the feel for DnD